Source/Scene/BillboardCollection.js

```diff
@@ -1218,6 +1237,9 @@ define([
                 if (this._shaderPixelOffsetScaleByDistance) {
                     vs.defines.push('EYE_DISTANCE_PIXEL_OFFSET');
                 }
+                if (defined(this._globe)) {
```

Comment:
Would we ever want this just for the depth test? If a user manually places a billboard, the regular depth test is turned on and the terrain depth isn't cleared, parts of a billboard might be under terrain.